### ISSUE TITLE
Disable shaders for RTG modes only

### ISSUE THREAD

**Issue by parkamonster** [FrodeSolheim/fs-uae#264]
I like to use CRT shaders for WHDLoad games, but I don't want them for RTG graphics. Would it be possible to make them only activate when using PAL/NTSC 15Khz, kind of like how you have a separate scanline option for normal graphics and RTG graphics? Thanks!

**Comment by FrodeSolheim** [comment]
Yes, this makes sense. I've had it in mind since the quite early days of shader support for FS-UAE. Maybe time to do something about it soon :)

**Comment by parkamonster** [comment]
That’s great to hear! Thanks
